blueocean-web/package.json

```diff
@@ -8,7 +8,8 @@
     "test": "gulp test",
     "test:watch": "gulp test:watch",
     "bundle": "gulp bundle",
-    "bundle:watch": "gulp bundle:watch"
+    "bundle:watch": "gulp bundle:watch",
+    "postinstall": "node ../bin/cleanshrink.js"
```

Comment:
Caching is configured through `Jenkinsfile` which fetches a npmrc file from the CI server. The node part here does not do any caching, it's just to clean up resolved URL of packages from shrinkwrap file. When the resolved URLs are in shrinkwrap file, it's not possible to use a cache server defined in npmrc.